src/test/kotlin/lotto/domain/LottosTest.kt

```diff
@@ -39,15 +28,17 @@ internal class LottosTest {
     )
 
     @Test
-    fun `match 테스트`() {
+    fun `match count 테스트`() {
         val lottos = Lottos(listOf(lotto1, lotto2))
-        val match = lottos.match(winningLotto)
+        val winningLotto = WinningLotto(lotto1, LottoNumber.of(7))
+
+        val result = lottos.match(winningLotto)
 
-        val first = Rank.of(winningLotto.matchCount(lotto1))
-        val third = Rank.of(winningLotto.matchCount(lotto2))
+        val first = winningLotto.match(lotto1)
+        val third = winningLotto.match(lotto2)
 
-        assertThat(match.elements.keys).contains(first, third)
-        assertThat(match.elements[first] ?: throw NullPointerException()).isEqualTo(1)
-        assertThat(match.elements[third] ?: throw NullPointerException()).isEqualTo(1)
+        assertThat(result.elements.keys).contains(first, third)
+        assertThat(result.elements[first] ?: throw NullPointerException()).isEqualTo(1)
+        assertThat(result.elements[third] ?: throw NullPointerException()).isEqualTo(1)
```

Comment:
해당 요소가 null이면 assertj에서 알려주니 우리가 직접 Exception을 던져주지 않아도 됩니다.

```suggestion
        assertThat(result.elements[first]).isEqualTo(1)
        assertThat(result.elements[third]).isEqualTo(1)
```